controllers/userController.js

```diff
@@ -183,11 +185,22 @@ const verifyResetPasswordToken = async (req, res) => {
     } else {
       res.status(200).send({ message: "user password reset token expired" })
     }
-
-
   } catch (err) {
     res.status(500).send(err)
   }
-
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses, forgotPassword, verifyResetPasswordToken }
+const uploadProfile = async (req, res) => {
+  const username = req.params.username;
+  try {
+    const userData = await userModel.findOne({ where: { username } })
```

Comment:
Here variable name here you are finding a user so take a variable name relatable.